notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb

```diff
@@ -241,7 +241,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
    "language": "python",
    "name": "python3"
```

Comment:
distributed training needs PySpark kernel too, no?

Reply:
It also needs it @jimdowling

Reply:
Ok, will fix.

Reply:
Fixed.